contrib/cirrus/setup_environment.sh

```diff
@@ -130,14 +115,24 @@ case "$OS_RELEASE_ID" in
             msg "Enabling container_manage_cgroup"
             setsebool container_manage_cgroup true
         fi
+
+        # For release 36 and later, use netavark/aardvark unless upgrade
+        # testing, since upgrading from CNI is not supported.
+        if [[ "$OS_RELEASE_VER" -ge 36 ]] && \
```

Comment:
Any way to stuff the value `36` into a variable in the .cirrus.yml that could be referenced here?  The more places we have to update a hard-coded value, the deeper my twitches get.

Reply:
Typically I agree with you, however in this case I'm pretty sure we'll never be going back to CNI in future Fedora releases.  That said, let me go over all the scripts here quickly and see if there's some other place(s) you're suggestion may apply...

Reply:
...Found one other thing that's F36-specific, but it's in a TODO comment, so I'm leaving it alone for now.  I did re-write the 119-120 comment to be more clear about the situation.  I'll hang onto that for my next push.